FHEM/00_SIGNALduino.pm

```diff
@@ -3847,868 +3847,1004 @@ sub SetSens {
+    </ul><br>
+    <i><u>note model radino:</u></i>
+    <ul>
+      <li>Sometimes there can be problems flashing radino on Linux. <a href="https://wiki.in-circuit.de/index.php5?title=radino_common_problems">Here in the wiki under point "radino & Linux" is a patch!</a>
+      </li>
+      <li>If the Radino is defined in this way <code>/dev/ttyACM0</code>, the flashing of the firmware should be done automatically. If this fails, the boot loader must be activated manually:
+      </li>
+      <li>To activate the bootloader of the radino there are 2 variants.
+        <ul>
+          <li>1) modules that contain a BSL-button:
+            <ul>
+              <li>apply supply voltage</li>
+              <li>press & hold BSL- and RESET-Button</li>
+              <li>release RESET-button, release BSL-button</li>
+              <li>(repeat these steps if your radino doesn't enter bootloader mode right away.)</li>
+            </ul>
+          </li>
+          <li>2) force bootloader:
+            <ul>
+              <li>pressing reset button twice</li>
+            </ul>
+          </li>
+        </ul>
+        In bootloader mode, the radino gets a different USB ID. This must be entered in the "flashCommand" attribute.<br>
+        If the bootloader is enabled, it signals with a flashing LED. Then you have 8 seconds to flash.
+      </li>
+    </ul>
+  </li><br>
+
+  <a name="raw"></a>
+  <li>raw<br>
+    Issue a SIGNALduino firmware command, without waiting data returned by
+    the SIGNALduino. See the SIGNALduino firmware code  for details on SIGNALduino
+    commands. With this line, you can send almost any signal via a transmitter connected
+
+    To send some raw data look at these examples:
+    P<protocol id>#binarydata#R<num of repeats>#C<optional clock>   (#C is optional)<br>
+    <br>Example 1: <code>set sduino raw SR;R=3;P0=500;P1=-9000;P2=-4000;P3=-2000;D=0302030</code>  sends the data in raw mode 3 times repeated
+    <br>Example 2: <code>set sduino raw SM;R=3;P0=500;C=250;D=A4F7FDDE</code>  sends the data manchester encoded with a clock of 250uS
+    <br>Example 3: <code>set sduino raw SC;R=3;SR;P0=5000;SM;P0=500;C=250;D=A4F7FDDE</code>  sends a combined message of raw and manchester encoded repeated 3 times
+    <ul><br>
+      <b>note: The wrong use of the upcoming options can lead to malfunctions of the SIGNALduino!</b><br><br>
+
+      <u>ONLY for DEBUG | <small>commands are dependent from firmware version!</small></u><br>
+      <li>CED -> Debug output on</li>
+      <li>CDD -> Debug output off</li>
+      <li>CDL -> LED off</li>
+      <li>CEL -> LED on</li>
+      <li>CER -> turn on data compression (config: Mred=1)</li>
+      <li>CDR -> disable data compression (config: Mred=0)</li>
+      <li>CSmscnt=[Wert] -> repetition counter for the split of MS messages</li>
```

Comment:
Das Kommando bietet die Firmware nicht an.

Die uC Befehle gehören hier auch irgendwie nicht in das Modul finde ich.

Reply:
>Befehle sind abh&auml;nging vom Firmwarestand!

Sagt doch alles. Dies resultiert daraus, das wir damals diese Dinge eingepflegt haben weil verschiedene Versionen.
Herausnehmen?